src/__configuration__/landingPage/cardData.ts

```diff
@@ -49,7 +49,7 @@ export const cardData = [
         title: 'GitHub',
         description: `All Brightlayer UI resources and materials are open source and available for viewing on our GitHub organization.`,
```

Comment:
This may need a little rewording since we aren't a GitHub organization anymore...maybe just "on GitHub."